### ISSUE TITLE
[Suggestion] add capsolver for captcha

### ISSUE THREAD

**Issue by PoloNX** [FuckingToasters/discord-phone-verifier#26]
Hi, capsolver is very very good and less flagged than others

**Comment by FuckingToasters** [comment]
> Hi, capsolver is very very good and less flagged than others

already exist for months

**Comment by PoloNX** [comment]
I don't fond it in plugins/captcha

**Comment by PoloNX** [comment]
> > Hi, capsolver is very very good and less flagged than others
> 
> already exist for months

this one ? https://www.capsolver.com/

**Comment by FuckingToasters** [comment]
> > > Hi, capsolver is very very good and less flagged than others
> > 
> > 
> > already exist for months
> 
> this one ? https://www.capsolver.com/

Yes, it's already added. at one point thier name was captchaai and then they changed name and rebranded their service. the filename is still captchaai but if you take a look inside that file, you'll notice, that it is using capsolver.com. However keep in mind that capsolver might not work as intented because discord is flagging their solutions. I suggest you to start using https://capmonster.cloud
